### ISSUE TITLE
Line highlighting

### ISSUE THREAD

**Issue by amannn** [kufii/CodeSnap#153]
Being able to generate line highlights like these would be very helpful:

![Screenshot 2024-06-25 at 12 47 38](https://github.com/kufii/CodeSnap/assets/4038316/f17b51d5-9206-4a56-966f-eca0cccd6e69)

**Comment by ArthurLobopro** [comment]
@amannn I think this extension in no more receiving features (the last commit was 3 years ago). I made a Codesnap based extension called Easy Codesnap with more features and fixes. If you want, take a look:

https://marketplace.visualstudio.com/items?itemName=ArthurLobo.easy-codesnap

**Comment by amannn** [comment]
Hey @ArthurLobopro, thank you for your reply here! Would be happy to switch, but how can I highlight specific lines with your extension? I gave it a go, but couldn't figure out how.

**Comment by ArthurLobopro** [comment]
@amannn Yes you can. You just need to click on the line number. There are 3 highlight levels as you can see here:

![image](https://github.com/user-attachments/assets/93ec89f0-fa17-4b90-a051-f73edcf0bfd7)

**Comment by amannn** [comment]
Oh cool, I've missed that—works like a charm!

One thing I'd personally like to customize is to turn off the color of the highlight marker on the left (or set it to `transparent`). Just so the result is a bit more minimalistic.

As a side note, I think you could also consider setting the default for the saving scale to 2x.

Other than that 💯, will be using this from now on! 👌

**Comment by ArthurLobopro** [comment]
@amannn Thank you! If you could, leave a review on extension marketplace or just tell your friends about my extension. There are a lot of variants and people barely know my variant.

One more thing. There are some hidden resources like edit title or watermark just by double clicking it, I hope this can be useful for you!
